src/test/java/uk/gov/pay/api/service/CreateAgreementServiceTest.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.pay.api.service;
+
+import org.apache.http.HttpStatus;
+import org.junit.Before;
+import org.junit.Test;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import uk.gov.pay.api.agreement.model.AgreementResponse;
+import uk.gov.pay.api.agreement.model.CreateAgreementRequest;
+import uk.gov.pay.api.agreement.model.builder.AgreementResponseBuilder;
+import uk.gov.pay.api.agreement.service.AgreementService;
+import uk.gov.pay.api.auth.Account;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+public class CreateAgreementServiceTest {
```

Comment:
Can we make this JUnit 5? Looks straightforward so the conversion should be easy.